content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -17,24 +17,15 @@ weight: 40
 
 For clarity, this guide defines the following terms:
 
-Node
-: A worker machine in Kubernetes, part of a cluster.
-
-Cluster
-: A set of Nodes that run containerized applications managed by Kubernetes. For this example, and in most common Kubernetes deployments, nodes in the cluster are not part of the public internet.
-
-Edge router
-: A router that enforces the firewall policy for your cluster. This could be a gateway managed by a cloud provider or a physical piece of hardware.
-
-Cluster network
-: A set of links, logical or physical, that facilitate communication within a cluster according to the Kubernetes [networking model](/docs/concepts/cluster-administration/networking/).
-
-Service
-: A Kubernetes {{< glossary_tooltip term_id="service" >}} that identifies a set of Pods using {{< glossary_tooltip text="label" term_id="label" >}} selectors. Unless mentioned otherwise, Services are assumed to have virtual IPs only routable within the cluster network.
+* Node: A worker machine in Kubernetes, part of a cluster.
```

Comment:
Lines 20 - 33 are set up as an inlined glossary.

Reply:
I checked the [documentation content guide](https://kubernetes.io/docs/contribute/style/content-guide/), I didn't see any guide about inline glossary suggestion, instead, some pages like [this](https://kubernetes.io/docs/tutorials/services/source-ip/#terminology), it list terms by markdown list, which is more clear to our users.

The first time I saw this page, I got an inconsistent feeling about this part, so I'd like to change this part in this PR.

Reply:
Hello @Colstuwjx .
I agree that this page may be the only page with a Definition List. Would the rendered list be acceptable to you if styled? The styling for the Definition List could be improved (indent the `<dd>` element, add a top margin for the `<dl>` element, use a quieter font for the `<dd>` element, update font size). I made a few quick changes as a test:


![image](https://user-images.githubusercontent.com/1696019/74039139-5b5a0a80-498f-11ea-8408-dfef960ab62d.png)

Reply:
@kbhawkey  Could you please share your suggested change in comment? So that I could also confirm about this change. BTW, if it looks hacky to do this change, I'd like to keep my PR suggested way, since we still don't have a official content style about this, adding a hacky way should be problematic.

Thanks!

Reply:
Hello @Colstuwjx,
I create PR #19034 as an example of styling options for definition lists (definitely not hacky 😄 ).

Reply:
@kbhawkey I have been checked this change locally, it LGTM!
Shall we take this change in this PR after we all agreed in #19034?